src/lib/modernExtend.ts

```diff
@@ -44,6 +44,46 @@ async function setupAttributes(
     }
 }
 
+export function temperature(): ModernExtend {
+    return {
+        fromZigbee: [fz.temperature],
+        exposes: [e.temperature()],
+        configure: async (device: Zh.Device, coordinatorEndpoint: Zh.Endpoint, logger: Logger) => {
+            await setupAttributes(device, coordinatorEndpoint, 'msTemperatureMeasurement', [{attribute: 'measuredValue'}], logger);
+        },
+        isModernExtend: true,
+    };
+}
+
+export function pressure(): ModernExtend {
+    return {
+        fromZigbee: [fz.pressure],
+        exposes: [e.pressure()],
+        configure: async (device: Zh.Device, coordinatorEndpoint: Zh.Endpoint, logger: Logger) => {
+            await setupAttributes(device, coordinatorEndpoint, 'msPressureMeasurement', [{attribute: 'measuredValue'}], logger);
+        },
+        isModernExtend: true,
+    };
+}
+
+export function humidity(): ModernExtend {
+    return {
+        fromZigbee: [fz.humidity],
+        exposes: [e.humidity()],
+        configure: async (device: Zh.Device, coordinatorEndpoint: Zh.Endpoint, logger: Logger) => {
+            await setupAttributes(device, coordinatorEndpoint, 'msRelativeHumidity', [{attribute: 'measuredValue'}], logger);
+        },
+        isModernExtend: true,
+    };
+}
+
+export function identify(): ModernExtend {
+    return {
+        toZigbee: [tz.identify],
```

Comment:
Can you add an expose here

Reply:
Not sure which one expose option you mean. Could you please elaborate a bit?

Reply:
For e.g. `export function humidity(): ModernExtend {` there is an `exposes: [e.humidity()],` returned, we should do the same here.

Reply:
Is there such expose for identify? I quickly searched for it, but unfortunately didn't find any related to it.

Reply:
A new one can be created for it, I propose to use `e.enum()`

Reply:
Sorry, I am not sure which state would you propose to add there. As this is just a command that will be sent to produce temporary device state.

Or do you mean to also add a reporter that will get `identifyTime` attribute and map the attribute value?